src/services/document-service.spec.js

```diff
@@ -291,7 +288,8 @@ describe('document-service', () => {
       });
 
       it('creates the revisions', async () => {
-        const createdRevisions = await db.documentRevisions.find({ key: documentKey }).toArray();
+        const createdRevisions = await db.documentRevisions.find({ key: documentKey }).sort({ order: 1 }).toArray();
```

Comment:
This test was unstable, it was getting the wrong ordering on the revisions and failing.

Reply:
It actually had the sort there exactly cause it was unstable otherwise (plus it should have been there anyway, like we have it in the service when it returns documents). Don't get when it vanished..

Reply:
:+1: